packages/commerce-sdk-react/src/auth/storage.test.ts

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright (c) 2022, Salesforce, Inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import {BaseStorage, ServerStorage} from './storage'
+
+const {window} = global
```

Comment:
Does jest inject this somehow? In basic node env, `window` will be `undefined`.

Reply:
In the `pwa-kit-dev` module, you'll see that the environment that we are using for jest is `jsdom` in the `/src/config/jest/jest.config.js` file. 
Whether or not this is the correct environment for be defaulted to for this package is a question that we can think about, but maybe we can take that outside of the scope of this PR.